t/t0013-config-file.t

```diff
@@ -260,6 +263,38 @@ test_expect_success 'start size=3 instance with ipc:// and custom topology' '
 	test_cmp conf8a.exp conf8a.out
 '
 
+waitgrep() {
+	local pattern=$1
+	local file=$2
+	local iter=$3
+	while test $iter -gt 0 && ! grep "$pattern" $file 2>/dev/null; do
+		sleep 0.3
+		iter=$(($iter-1))
+	done
+	test $iter -gt 0 && return 0
+	return 1
+}
```

Comment:
Would `t/scripts/waitfile.lua` work here?

Reply:
That's what I started with but it seems to not work without a connection to the broker?  (That surprised me - maybe I've got it wrong)

Reply:
Oh, wow, you're right. Although it only needs the handle for the reactor, that's too bad (haven't looked at it in awhile.)